system_tests/vision.py

```diff
@@ -474,13 +474,13 @@ def tearDown(self):
             value.delete()
 
     def _assert_color(self, color):
-        self.assertIsInstance(color.red, int)
-        self.assertIsInstance(color.green, int)
-        self.assertIsInstance(color.blue, int)
+        self.assertIsInstance(color.red, (int, float))
+        self.assertIsInstance(color.green, (int, float))
+        self.assertIsInstance(color.blue, (int, float))
```

Comment:
Cast to floats https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2937/commits/a39be5fe5a9be88de9cf501856589d1965a3e409